py/mkenv.mk

```diff
@@ -57,7 +57,8 @@ MAKE_MANIFEST = $(PYTHON) $(TOP)/tools/makemanifest.py
 MAKE_FROZEN = $(PYTHON) $(TOP)/tools/make-frozen.py
 MPY_TOOL = $(PYTHON) $(TOP)/tools/mpy-tool.py
 
-MPY_LIB_DIR = $(TOP)/../micropython-lib
+MPY_LIB_SUBMODULE_DIR = $(TOP)/lib/micropython-lib
+MPY_LIB_DIR = $(MPY_LIB_SUBMODULE_DIR)
```

Comment:
Should this be `?=` so it can be overridden as a command line arg?

Reply:
`?=` is only needed to allow overriding using an environment variable (`MPY_LIB_SUBMODULE_DIR=... make`) but `=` allows overriding from a command line arg (`make MPY_LIB_SUBMODULE_DIR=...`).

https://www.gnu.org/software/make/manual/html_node/Overriding.html#Overriding

Reply:
oh... learn something new every day, thanks @dlech :-D